src/core_plugins/kibana/public/dev_tools/directives/dev_tools_app.js

```diff
@@ -0,0 +1,18 @@
+import uiModules from 'ui/modules';
+import devTools from 'ui/registry/dev_tools';
+import template from 'plugins/kibana/dev_tools/partials/dev_tools_app.html';
+import 'plugins/kibana/dev_tools/styles/dev_tools_app.less';
+
+uiModules
+.get('apps/dev_tools')
+.directive('kbnDevToolsApp', function (Private, $location) {
+  return {
+    restrict: 'E',
+    template,
+    transclude: true,
```

Comment:
If you add `replace: true` to this directive definition, I think you can do away with the LESS selector on this directive's element:

```less
kbn-dev-tools-app {
  .flex-parent; // no longer needed
}